docs/topics/class-based-views/generic-display.txt

```diff
@@ -419,12 +419,12 @@ object -- so we simply override it and wrap the call::
 
         def get_object(self):
             # Call the superclass
-            object = super().get_object()
+            result = super().get_object()
             # Record the last accessed date
-            object.last_accessed = timezone.now()
-            object.save()
+            result.last_accessed = timezone.now()
+            result.save()
             # Return the object
-            return object
+            return result
```

Comment:
We use `obj` to avoid the `object` collision in the code base.

https://github.com/django/django/blob/66b6b689239dad3f017d2a3495df748cbee5debb/django/views/generic/detail.py#L50-L56

Reply:
@morenoh149 You can also find and change `object`variable in "_topics/http/shortcuts.txt_" or "_topics/class-based-views/generic-display.txt_".